crates/assistant2/Cargo.toml

```diff
@@ -73,6 +73,7 @@ util.workspace = true
 uuid.workspace = true
 workspace.workspace = true
 zed_actions.workspace = true
+file_icons.workspace = true
```

Comment:
Could we keep this list of dependencies sorted alphabetically?

Reply:
sure!

Reply:
Fixed in https://github.com/zed-industries/zed/pull/22928/commits/eebb114097d9959a54ee58652abaa7e49db48754